opensrp-app/src/main/java/org/smartregister/view/fragment/BaseRegisterFragment.java

```diff
@@ -82,7 +84,11 @@ public void beforeTextChanged(CharSequence charSequence, int i, int i2, int i3)
 
         @Override
         public void onTextChanged(final CharSequence cs, int start, int before, int count) {
-            filter(cs.toString(), "", getMainCondition(), false);
+            org.smartregister.Context opensrpContext = CoreLibrary.getInstance().context();
+          if(opensrpContext.getAppProperties().hasProperty(AllConstants.PROPERTY.ENABLE_FAST_SEARCH) && !opensrpContext.getAppProperties().getPropertyBoolean(AllConstants.PROPERTY.ENABLE_FAST_SEARCH))
```

Comment:
Instead of using both helper methods `hasProperty(` and `getProperty(` you can just use the `isTrue(` helper method since it combines the two